driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1448,119 +1448,125 @@ private Manager(
     // on it so synchronized is good enough.
     synchronized void init() {
       checkNotClosed(this);
-      if (isInit) return;
+      if (isInit) {
+        if (initException == null) {
```

Comment:
It would be better to move this logic to `checkNotClosed`, as that is used in a number of places and offers the opportunity to notify the user that there was an error during initialization (which is more specific than cluster was closed)